docs/docsite/rst/dev_guide/developing_collections_path.rst

```diff
@@ -0,0 +1,160 @@
+* `Ansible Automation Hub <https://www.ansible.com/products/automation-hub>`_ if you have the Ansible Automation Platform subscription
+
+In case the solutions you found are not fully sufficient or have flaws, consider improving them rather than creating your own.
+
+If you already have your content written and used in your workflows, you could think of integrating it to the existing solutions.
+However, if they have significant flaws, licensing or major design issues, their corresponding projects are unmaintained, or if you think your implementation will work better, you are encouraged to create and share your own one.
+
+.. _create_content:
+
+Create your content
+===================
+
+You :ref:`tried <examine_existing_content>` but have not found any sufficient solution for your automation issue.
+
+Use one of the following guides:
+
+* :ref:`Roles guide<playbooks_reuse_roles>`: if you want to create a role.
+* :ref:`Developer guide<developer_guide>`: if you want to create a new Ansible module or plugin.
+
+Put your content in a collection
+================================
+
+You :ref:`created <create_content>` new content.
+
+Now it is time to create a collection to share your work with the community.
+Use the :ref:`Developing collections guide<developing_collections>` to learn how.
+
+We recommend you to use the `collection_template repository <https://github.com/ansible-collections/collection_template>`_ as a basis for your collection.
+
+Write good user collection documentation
+========================================
+
+Your collection's ``README.md`` file contains a quick-start installation and usage guides.
+You can use the `community.general collection README file <https://github.com/ansible-collections/community.general/blob/main/README.md>`_ as an example.
+
+If your collection contains modules or plugins, make sure their documentation is comprehensive.
+Use the :ref:`Module format and documentation guide<developing_modules_documenting>` and :ref:`Ansible documentation style guide<style_guide>` to learn more.
+
+Publish your collection source code
+===================================
+
+Publish your collection on a platform for software development and version control such as `GitHub <https://github.com/>`_.
+
+It can be your personal repository or your organization's one.
+You can also `request <https://github.com/ansible-collections/overview/issues>`_ a repository under the `ansible-collections <https://github.com/ansible-collections/>`_ organization.
+
+Make sure your collection contains exhaustive license information.
+Ansible is an open source project, so we encourage you to license it under one of open source licenses.
+If you plan to submit your collection for inclusion in the Ansible community package, your collection must satisfy the `licensing requirements <https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html#collection-licensing-requirements>`_.
+
```

Comment:
replace with a :ref: to the anchor at that file location.  Even if that file isn't part of the core docs, CI should still work as the intersphinx mapping should pick it up..

Reply:
ah, missed it while moving from the community-docs repo, fixed